app/controllers/dams_objects_controller.rb

```diff
@@ -4,7 +4,7 @@ class DamsObjectsController < DamsResourceController
   def check_type( document )
     # redirect to collection path if it is a type of collection record
     arr_of_col = [ "DamsAssembledCollection",
-                   "DamsProvenanceCollection", 
+                   "DamsProvenanceCollection",
```

Comment:
Another, ignore the "Ignoring". Despite this being preexisting code, I went ahead and updated it to use single-quoted strings because Hound was acting up.